.travis.yml

```diff
@@ -0,0 +1,19 @@
+sudo: required
+
+services:
+  - docker
+
+language: python
+
+python:
+  - "2.7"
+
+before_install:
+  # Download Cromwell
+  - wget -O $TRAVIS_BUILD_DIR/cromwell-42.jar https://github.com/broadinstitute/cromwell/releases/download/42/cromwell-42.jar;
+
+install:
+  - pip install cwl-runner cwltool==1.0.20160325210917 schema-salad==1.7.20160316203940 avro==1.9.0
```

Comment:
Let's match the version of cwltool at https://dockstore.org/api/metadata/runner_dependencies?client_version=1.6.0&python_version=2

Double check that cromwell matches the version that the CLI installs as well

Reply:
Okay cool

Reply:
I just updated it to python3.6 and newest versions of each requirement